tasks/libs/issue/model/actions.py

```diff
@@ -0,0 +1,117 @@
+from time import sleep
+
+from tasks.libs.ciproviders.github_api import GithubAPI
+from tasks.libs.issue.assign import assign_with_rules
+from tasks.libs.issue.model.constants import BASE_MODEL, MODEL, TEAMS
+
+
+def generate_model():
+    gh = GithubAPI('DataDog/datadog-agent')
+    d = gh.repo
+    issues = []
+    teams = []
+    n = 0
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏You can use `enumerate` instead of this

Reply:
Done